test/e2e/specs/all-features-confirmation/record/presentation.spec.js

```diff
@@ -163,4 +170,92 @@ describe('View existing record,', function() {
         });
     });
 
+    describe("For side panel table of contents in Record App", function() {
+
+        beforeAll(function() {
+            var url = browser.params.url + "/record/#" + browser.params.catalogId + "/" + testParams.sidePanelTest.schemaName + ":" + testParams.sidePanelTest.tableName +  "/id=" + testParams.sidePanelTest.id;
+            browser.get(url);
+            recSidePan = chaisePage.recordPage.getElementById('recordSidePan-5');
+            fiddlerBtn = element(by.className('sidePanFiddler')).element(by.tagName('i')),
+            chaisePage.waitForElement(fiddlerBtn);
+        });
+        it('Table of contents should be collapsed by default', function(){
+            var recordSidePan = chaisePage.recordPage.getElementById('record-side-pan');
+            recordSidePan.isDisplayed().then(function (bool) {
+                expect(bool).toBeFalsy();
+            });
+        });
+
+        it('Table of contents should match all related table', function(){
+            var  allPan = element.all(by.xpath("//div[contains(@id,'recordSidePan')]"));
```

Comment:
Please write this as a `chaisePage.recordPage` function. This can be done with a statement similar to:
`element.all(by.css('.some-class-you-added')` 
Or use a parent element that you can fetch and get all children of that parent:
`element(by.css('faceting-container)).all(by.tagName('li'))`